public/js/main.js

```
@@ -7,6 +7,27 @@ $(function () {
     }
   })
 
+  // Default value keeps the old behaviour
+  var timeFormatting = {
+    type : 'function', // function or format
+    value: 'fromNow' // fromNow/toISOString if type is function, or moment format value if type is format (see Moment.js doc)
+  }
+  // e.g {type: 'function', value: 'fromNow'}
+  //     {type: 'function', value: 'toISOString'}
+  //     {type: 'format', value: 'LLL'}
+  //     {type: 'format', value: 'DD/MM/YYYY HH:mm:ss'}
+
+  function formatDate(date) {
+    var mDate = moment(date)
+    if ((timeFormatting.type === 'function') && ((typeof mDate[timeFormatting.value]) === 'function')) {
+      return mDate[timeFormatting.value]()
+    } else if (timeFormatting.type === 'format') {
+      return mDate.format(timeFormatting.value)
+    } else {
+      return mDate.fromNow() // Old behaviour in case of invalid data
```

Maybe we should throw an error here instead of silently ignoring it?